remix/remix-lib/src/execution/execution-context.js

```diff
@@ -292,8 +292,8 @@ var transactionDetailsLinks = {
 }
 
 var transactionDetailsLinksRV = {
-  '13137357': 'https://kevm-testnet.iohkdev.io/transaction/',
```

Comment:
Hi @krisajenkins , I think this will increase the difficulty of developing Remix IDE locally.
I am thinking that maybe we could do a check like:
`location.hostname.match(/^(localhost|127\.0\.0\.1)$/) ? absolute_url : "/transaction/"`

Reply:
So If Remix is running locally, we use absolute URLs?

Reply:
@krisajenkins Sorry I am wrong, for this part line `295` and `296` I think we should just use absolute URLs.